jlibra-core/src/main/java/dev/jlibra/admissioncontrol/transaction/result/SubmitTransactionException.java

```diff
@@ -0,0 +1,11 @@
+package dev.jlibra.admissioncontrol.transaction.result;
+
+import dev.jlibra.LibraRuntimeException;
+
+public class SubmitTransactionException extends LibraRuntimeException {
```

Comment:
I think this should be checked exception because client of SubmitTransactionResult. fromGrpcObject, most likely, would like to handle success and failure. Since there is either success or exception then declaring exception on the method definition would help handling failure. What do you think?

Reply:
I think you're right, the exception needs to be handled always so it makes sense to make it checked